src/main/java/racingcar/domain/MoveFactory.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain;
+
+public class MoveFactory {
+
+    private final NumberGenerator numberGenerator;
+
+    public MoveFactory(final NumberGenerator numberGenerator) {
+        this.numberGenerator = numberGenerator;
+    }
+
+    public boolean isMove() {
+        int randomNumber = numberGenerator.generate();
+        if (isOverOrFour(randomNumber)) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    public boolean isMove() {
        return isOverOrFour(numberGenerator.generate());
    }
```

boolean 을 리턴받아서, boolean 이 true 면 true 를 리턴하고, false 면 false 를 리턴하는 것은 자명한 일을 반복하는 행위라고 생각합니다 :)